recipe/meta.yaml

```diff
@@ -24,13 +24,17 @@ requirements:
     - gcc  # [osx]
     - boost 1.63.*  # note this is intentionally NOT the latest version; synced with casacore which has special needs
     - casacore 2.2.*
-    - numpy x.x
+    - numpy 1.7.*  # [py27]
+    - numpy 1.9.*  # [py35]
+    - numpy 1.11.*  # [py36]
```

Comment:
The trick here is to use the oldest possible `numpy` to build. I did not find `casacore` lower limit in the repo but `1.8` is usually safe.

Also, after I wrote the docs I submitted `numpy 1.8` and `numpy 1.9` updates to all Pythons, so we are not limited to `numpy 1.9.*  # [py35]` and `numpy 1.11.*  # [py36]`. The only problem is that we don't have Windows packages for those yet, so `py36`+`win ` is still limited to `numpy 1.11.*  # [py36]`.